src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  border: solid 1px #f3f3f3;
+  width: $card-width;
+  height: $card-height;
+
+  background-image: url($prod-img-source);
+  background-position-y: 32px;
+  background-position-x: 19px;
+  background-size: $prod-img-size;
+  background-repeat: no-repeat;
+
+  &__text {
+    line-height: 18px;
+    font-size: $font-size-default;
+    font-weight: 500;
+    color: #060b35;
+
+    margin-top: 206px;
+    padding: 0 16px;
+  }
+
+  &__prod-code {
+    color: #616070;
+    font-size: 10px;
+    padding: 4px 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    margin-top: 16px;
+    padding: 0 16px;
+
+    &-text {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+
+      margin-left: 17px;
+    }
+  }
+
+  &__price {
+    font-size: $font-size-default;
+    font-weight: 400;
+    line-height: 18px;
+    color: #616070;
+
+    margin-top: 24px;
+    padding: 0 16px;
+    display: flex;
+    flex-direction: row;
+
+    &-value {
+      margin-left: 85px;
+      font-weight: 700;
+      font-size: 16px;
+      color: #000;
+    }
+  }
+
+  &__button-buy {
+    width: $button-width;
+    height: $button-height;
+
+    border-width: 0;
+    border-radius: 5px;
+    background-color: $button-color;
+    margin: 16px;
+
+    color: $button-font-color;
+    font-size: 14px;
+    font-weight: 700;
+    text-align: center;
+  }
+}
+
+.card:hover {
+  .card__button-buy {
+    background-color: $button-color--hover;
+    color: $button-font-color--hover;
+    border: $button-border-color--hover;
```

Comment:
The 'border' property is being set to a color variable, which is incorrect. The 'border' property should include border width, style, and color. If you intend to change only the border color on hover, use 'border-color' instead.